server/routes/checkoutRoutes/checkout.routes.js

```diff
@@ -0,0 +1,8 @@
+const {webhook, createCheckoutSession, createSubscriptionPayment} = require('../../controllers/checkoutController/checkout.controller.js');
+const express = require('express');
+
+module.exports = (app) => {
+  app.post('/checkout/createSession', createCheckoutSession); //create a checkout session for onetime purchase
```

Comment:
What params are required for these routes?

Reply:
You can see that in the comments under the endpoint implementation. I have yet to do the api docs for them as they're not finalized. 
Create checkout session takes an array of items, each with its price_id and the quantity.
Create subscription takes a single price_id.
Webhook we don't need to worry about as stripe calls this, not us, but basically they pass the event through.